lib/senkyoshi/models/content.rb

```diff
@@ -21,12 +21,19 @@ class Content < Resource
       "x-bb-syllabus" => "WikiPage",
     }.freeze
 
-    attr_accessor(:title, :body, :id, :files)
+    attr_accessor(:title, :body, :id, :files, :url)
     attr_reader(:extendeddata)
 
-    def self.from(xml, pre_data)
+    def self.from(xml, pre_data, resource_xids)
       type = xml.xpath("/CONTENT/CONTENTHANDLER/@value").first.text
       type.slice! "resource/"
+      xml.xpath("//FILES/FILE").map do |file|
```

Comment:
So some of the attachments weren't getting attached because the files they were referencing didn't exist in the files that were exported. So what this is doing is first I am getting the list of files that were exported from the code in the senkyoshi.rb. Then it's eventually passing this into this file as the resources xids, and then this is going through the files that are said to be in the content files and checking to see if they exist in the list of files. Then if they exist it does nothing and continues the same way, but if the file doesn't exist it turns the attachment into a document -- Wikipage. Then when it creates the wikipage it lets the user know that the file didn't exist with a note saying which files don't exist instead of not creating the attachment at all. Not sure if it's ideal, but if you have any suggestions let me know.